index.html

```diff
@@ -37,6 +37,9 @@
     ctx.scale(2, 2);
     ctx = canvas.getContext('2d');
   
+    const params = (new URL(document.location)).searchParams;
+    const speed = params.get("speed");
```

Comment:
can we get some validation on this? If someone passes something other than a number we might get some weird results

Reply:
https://gph.is/2QHZK7r